src/TMediaData.h

```diff
@@ -54,7 +54,18 @@ class TMediaData
     QString getMediaFileName() const { return mMediaFileName; }
     void setMediaFileName(QString mediaFileName) { mMediaFileName = mediaFileName; }
     int getMediaVolume() const { return mMediaVolume; }
-    void setMediaVolume(int mediaVolume) { mMediaVolume = mediaVolume; }
+    void setMediaVolume(int mediaVolume) {
+        if (mediaVolume == TMediaData::MediaVolumePreload) {
+            // Support preloading
+            mMediaVolume = TMediaData::MediaVolumePreload;
+        } else if (mediaVolume > TMediaData::MediaVolumeMax) {
+            mMediaVolume = TMediaData::MediaVolumeMax;
+        } else if (mediaVolume < TMediaData::MediaVolumeMin) {
+            mMediaVolume = TMediaData::MediaVolumeMin;
+        } else {
+            mMediaVolume = mediaVolume;
+        }
```

Comment:
Qt has a macro for this:
```suggestion
        } else {
            mMediaVolume =  qBound(TMediaData::MediaVolumeMin, mMediaVolume, TMediaData::MediaVolumeMax);
        }
```